[BUG] 4C not compiling on LNM clusters

## Bug Description: What's going wrong?
<!---
Tell us about the bug, its nasty behavior, its context, and its severity. When was the last time everything worked (date/time; SHA1s; etc.)? What did you do that made the bug rear its ugly head?

Include configure logs / build logs / error logs / screenshots / ... to provide as much useful information as possible.
-->

Since yesterday 4C no longer compiles on our clusters

see

https://gitlab.lrz.de/baci/baci-config-lnm/-/jobs/11759757

and 

https://gitlab.lrz.de/baci/baci-config-lnm/-/jobs/11759760.

The problem appears to be:

```
/home/gitlab-runner/builds/LfDka8JgZ/0/baci/baci-config-lnm/src/core/linalg/src/dense/4C_linalg_serialdensevector.hpp:16:10: fatal error: c++/13/span: No such file or directory
   16 | #include <c++/13/span>
      |          ^~~~~~~~~~~~~
compilation terminated.
```

which was introduced in #1297 .

@sebproell you probably know how to solve this faster than me 

## Expectations
<!---
What behavior is expected, such that this bug is considered fixed? How can this be tested to prevent this bug in the future?
- [ ] First do this.
- [ ] Then do that.
- [ ] Also this other thing.
-->

## Possible Solution
<!---
Not obligatory, but if you have any idea how to address this, please outline it here.
-->

## Steps to Reproduce
<!---
Provide a link to a live example, or an unambiguous set of steps to reproduce this issue. Include code to reproduce, if relevant.
1. Do this.
1. Do that.
1. Shake fist angrily at computer.
-->

## Your Environment
<!---
Include relevant details about your environment such that we can replicate this issue.
-->
- **Relevant repo SHA1s:**
- **Relevant configure flags or configure script:**
- **Operating system and version:**
- **Compiler and TPL versions:**

## Related Issues and Pull Requests
<!---
If applicable, let everybody know how this is related to any other open issues:
-->
- Blocks
- Follows
- Related to
- Part of
- Composed of

## Interested Parties
<!---
If there's any team or developer, who you think should be looped in on this bug report, feel free to @mention them here.
-->

--- a/src/core/linalg/src/dense/4C_linalg_serialdensevector.hpp
+++ b/src/core/linalg/src/dense/4C_linalg_serialdensevector.hpp
@@ -13,7 +13,7 @@
 
 #include <Teuchos_SerialDenseVector.hpp>
 
-#include <c++/13/span>
+#include <span>
 
 FOUR_C_NAMESPACE_OPEN
 

